src/lsp/cobol_ast/misc_descr.ml

```diff
@@ -655,7 +656,8 @@ let pp_select_clause ppf = function
   | SelectRecordKey { key; source } ->
     Fmt.pf ppf "RECORD %a%a"
       pp_qualname key
-      Fmt.(elist ~prefix:(any "SOURCE ") ~sep:sp pp_name') source
+      Pretty.(list ~fopen:"SOURCE " ~fsep:"@ " ~fclose:"" ~fempty:""
```

Comment:
Why `Pretty.(...)` instead of `(Pretty.list ...)` ?

Reply:
probably so that `pp_name'` (below) indents properly.